.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Extractors/ChineseDateExtractorConfiguration.cs

```diff
@@ -118,6 +124,72 @@ public List<ExtractResult> Extract(string text, DateObject referenceTime)
             return Token.MergeAllTokens(tokens, text, ExtractorName);
         }
 
+        public int GetYearFromText(Match match)

```

Comment:
This method would be the same as in Japanese, right?
Why not have it in a base CJK class and just call from here passing the right parser config?

Reply:
Is it also the same as in other languages? If so, we should decide where to place it for reuse.